reana-ui/src/pages/launchOnReana/Welcome.js

```diff
@@ -11,32 +11,39 @@ import { Link } from "react-router-dom";
 import { useSelector } from "react-redux";
 
 import { getConfig } from "~/selectors";
+
 import exampleRootImg from "~/images/example-root.png";
 import exampleAtlasRecastImg from "~/images/example-atlas-recast.png";
 import exampleCmsHiggsTauTauImg from "~/images/example-cms-htt.png";
 
 import styles from "./Welcome.module.scss";
 
-const DEMOS = [
+const DEFAULT_DEMO_REPOSITORIES = [
```

Comment:
Should we leave these here? In https://github.com/reanahub/reana/pull/746, I added the same repositories in the `values.yaml` file, mainly because it serves an example to admins that want to change them, whereas here the main purpose is to have a default option in case the `ui_launcher_repositories` value is removed. Do you think it is okay or does it feel like doing the same thing twice?

Reply:
Let's keep these also here